xml/day2_ui_documentation.xml

```diff
@@ -36,7 +36,9 @@
   was deployed. Access to this network is necessary to be able to access the
   &clm; Admin UI and log in. Depending on the network setup, it may be
   necessary to use an SSH tunnel similar to what is recommended in <xref
-  linkend="running_install_ui"/>.
+  linkend="running_install_ui"/>. The Admin UI requires Keystone and HAProxy
+  to be running and to be accesible. If Keystone or HAProxy are not running,
```

Comment:
Keystone -> `&o_ident;`; HAProxy -> &haproxy;

seems like Cloud should not be capitalized

Reply:
good catch, fixed in latest revision. thanks!